strings.js

```diff
@@ -0,0 +1,32 @@
+const sayHello = (word) => {
+  return ('Hello, ' + word + '!');
+};
+
+const uppercase = (string) => {
+  return string.toUpperCase();
+};
+
+const lowercase = (string) => {
+  return string.toLowerCase();
+};
+
+const countCharacters = (string) => {
+  return string.length; 
+};
+
+const firstCharacter = (string) => {
+  return string.charAt(0);
```

Comment:
You can also use square bracket notation to solve this with a bit less typing. `string[0]`